wls-eai-service/src/test/java/de/muenchen/oss/wahllokalsystem/eaiservice/rest/wahlvorschlag/WahlvorschlagControllerIntegrationTest.java

```diff
@@ -0,0 +1,252 @@
+
+            Assertions.assertThat(response.getResponse().getContentAsString()).isEmpty();
+        }
+
+        @Test
+        @WithMockUser(authorities = Authorities.SERVICE_LOAD_WAHLVORSCHLAEGELISTE)
+        @Transactional
+        void dataFound() throws Exception {
+            val wahlID = "wahlID";
+            val forDate = LocalDate.of(2024, 10, 10);
+
+            val wahlvorschlag1 = new Wahlvorschlag(1, "wahlvorschlag1", true, Set.of(
+                    new Kandidat("name1", 1, false, 1, false),
+                    new Kandidat("name2", 2, true, 2, true)));
+            val wahlvorschlag2 = new Wahlvorschlag(1, "wahlvorschlag2", true, Set.of(
+                    new Kandidat("name3", 1, false, 1, false),
+                    new Kandidat("name4", 2, true, 2, true)));
+            val wahlvorschlaege1 = new Wahlvorschlaege("wahlbezirkID1", "wahlID1", "stimmzettelgebietID1", Set.of(wahlvorschlag1, wahlvorschlag2));
+
+            val wahlvorschlag3 = new Wahlvorschlag(1, "wahlvorschlag1", true, Set.of(
+                    new Kandidat("name5", 1, false, 1, false),
+                    new Kandidat("name6", 2, true, 2, true)));
+            val wahlvorschlag4 = new Wahlvorschlag(1, "wahlvorschlag2", true, Set.of(
+                    new Kandidat("name7", 1, false, 1, false),
+                    new Kandidat("name8", 2, true, 2, true)));
+            val wahlvorschlaege2 = new Wahlvorschlaege("wahlbezirkID2", "wahlID1", "stimmzettelgebietID2", Set.of(wahlvorschlag3, wahlvorschlag4));
+
+            val wahlvorschlaegeListe = new WahlvorschlaegeListe(forDate, wahlID, Set.of(wahlvorschlaege1, wahlvorschlaege2));
+
+            val wahlvorschlaegeListeToLoad = wahlvorschlaegeListeRepository.save(wahlvorschlaegeListe);
+
+            val request = MockMvcRequestBuilders.get("/vorschlaege/wahl/" + wahlID + "/liste?forDate=" + forDate);
+
+            val response = api.perform(request).andExpect(status().isOk()).andReturn();
+            val responseBodyAsDTO = objectMapper.readValue(response.getResponse().getContentAsString(), WahlvorschlaegeListeDTO.class);
+
+            val expectedResponseDTO = wahlvorschlagMapper.toDTO(wahlvorschlaegeListeToLoad);
+
+            Assertions.assertThat(responseBodyAsDTO).isEqualTo(expectedResponseDTO);
+        }
+    }
+
+    @Nested
+    class LoadReferendumvorlagen {
+        @Test
+        @WithMockUser(authorities = Authorities.SERVICE_LOAD_REFERENDUMVORLAGEN)
+        void noDataFound() throws Exception {
+            val request = MockMvcRequestBuilders.get("/vorschlaege/referendum/wahlID/wahlbezirkID");
+
+            val response = api.perform(request).andExpect(status().isNotFound()).andReturn();
+
+            Assertions.assertThat(response.getResponse().getContentAsString()).isEmpty();
+        }
+    }
+
+    @Test
+    @WithMockUser(authorities = Authorities.SERVICE_LOAD_REFERENDUMVORLAGEN)
+    @Transactional
+    void dataFound() throws Exception {
```

Comment:
der Test sollte wohl in die Nested-Testsuit `LoadReferendumvorlagen `

Reply:
resolved with f64c1b483641dbd2e5e1cdb0e594d0f2baf68c21